metric/manager.go

```diff
@@ -29,11 +29,20 @@ var (
 		Help:      "Bucketed histogram of processing time (s) of requests.",
 		Buckets:   prometheus.ExponentialBuckets(0.1, 2, 8),
 	}, reqLabels)
+
+	resSize = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "response_size_bytes",
+		Help:      "Bucketed histogram of the HTTP response sizes in bytes.",
+		Buckets:   []float64{1000, 5000, 10000, 20000, 30000, 40000, 50000},
```

Comment:
Need suggestion here, what bucket range we see better for the response_data we deal with.  Here response size in bytes and I have kept bucket like 0-1000 bytes, 1001-5000 byte etc. In other words, ~1kb, ~5kb, ~10kb, ~20k, ~30k, ~40k, ~50k.  @aslakknutsen I need these bucket range from you.